src/pb_view.cpp

```diff
@@ -355,4 +351,26 @@ void pb_view::set_dllist_keymap_hint()
 	dllist_form.set("help", keymap_hint);
 }
 
+std::string pb_view::format_line(const std::string& podlist_format,
+	const download* dl,
```

Comment:
Since this function *requires* a `download` instance, I'd use reference instead of a pointer here.

Reply:
Got it.